dev-packages/application-package/src/application-props.ts

```diff
@@ -86,7 +86,8 @@ export namespace FrontendApplicationConfig {
         defaultIconTheme: 'theia-file-icons',
         electron: ElectronFrontendApplicationConfig.DEFAULT,
         defaultLocale: '',
-        validatePreferencesSchema: true
+        validatePreferencesSchema: true,
+        reloadOnReconnect: false
```

Comment:
I can see the reasoning, but if it can only improve the behavior of the app, why not set it as `true` by default? If you say there are possible drawbacks though, I can respect that. I just don't see any myself right now.

Reply:
If an adopter relies on this behavior, they will be broken by the change, if they do not, or even welcome the change, they can easily set the flag in their app. It's the "do no harm" principle, basically.